packages/svelte/src/internal/client/types.d.ts

```diff
@@ -307,15 +214,14 @@ export type Render = {
 	d: null | TemplateNode | Array<TemplateNode>;
 	/** effect */
 	e: null | Effect;
-	/** transitions */
-	s: Set<Transition>;
 	/** prev */
 	p: Render | null;
 };
 
 export type Raf = {
 	tick: (callback: (time: DOMHighResTimeStamp) => void) => any;
 	now: () => number;
+	tasks: Set<any>;
```

Comment:
general observation: everything in here (types and all their properties) should have jsdoc, it's where things come together on a type level and so this could help get up to speed with how things work together quicker